socorro/processor/signature_utilities.py

```diff
@@ -687,41 +687,32 @@ def _get_crashing_thread(self, processed_crash):
 
     #--------------------------------------------------------------------------
     def _action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
-        result = super(SignatureRunWatchDog, self)._action(
-            raw_crash,
-            raw_dumps,
-            processed_crash,
-            processor_meta
-        )
         processed_crash['signature'] = (
             "shutdownhang | %s" % processed_crash['signature']
         )
-        return result
+        return True
 
 
 #==============================================================================
-class SignatureShutdownTimeout(SignatureGenerationRule):
+class SignatureShutdownTimeout(Rule):
     """Customize the signature for shutdown timeouts"""
 
     def version(self):
         return '1.0'
 
     def _predicate(self, raw_crash, raw_dumps, processed_crash, proc_meta):
-        return 'AsyncShutdownTimeout' in raw_crash
+        try:
+            return bool(raw_crash['AsyncShutdownTimeout'])
+        except KeyError:
+            return False
 
     def _action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
-        result = super(SignatureShutdownTimeout, self)._action(
-            raw_crash,
-            raw_dumps,
-            processed_crash,
-            processor_meta
-        )
         parts = ['AsyncShutdownTimeout']
         try:
-            shutdown_data = ujson.loads(raw_crash['AsyncShutdownTimeout'])
+            shutdown_data = json.loads(raw_crash['AsyncShutdownTimeout'])
```

Comment:
I think we decided at some point to use ``json`` instead of ``ujson``, but I don't remember the reasoning. But looking through the ``processor`` module, it seems we mostly use ujson so I'll revert that.